.github/workflows/tests-django.yml

```diff
@@ -0,0 +1,27 @@
+name: Django tests
+
+on: push
+
+jobs:
+  tests-django:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v3
+
+      - name: Create environment file
+        run: cp .env.sample .env
+        working-directory: .devcontainer
+
+      - name: Build base image
+        run: docker compose build client
+        working-directory: .devcontainer
```

Comment:
I wanted to keep this as simple as possible (to start) so didn't bother with the fancier Buildx-based Docker config that other workflows have; we can switch to that now or later.